File: backend/Models/Book.cs
namespace backend.Models
{
    public class Book
    {
        public int id { get; set; }
        public string name { get; set; } = string.Empty;
        public string author { get; set; } = string.Empty;
        public string description { get; set; } = string.Empty;
        public bool is_borrowed { get; set; }
    }
}

File: frontend/src/types/Book.ts
export interface IBook {
    id: number;
    name: string;
    author: string;
    description: string;
    is_borrowed: boolean;
}

export interface ICreateBook {
    name: string;
    author: string;
    description: string;
}

export interface IYourBook {
    id: number;
    book_name: string;
    borrower_name: string;
    date_borrowed: Date;
}